src/asm_marshal.cpp

```diff
@@ -60,6 +60,8 @@ static uint8_t imm(Un::Op op) {
     case Op::LE32: return 32;
     case Op::BE64:
     case Op::LE64: return 64;
+    case Op::ZEXT32:
```

Comment:
I don't think ZEXT needs a new instruction, the existing BPF_MOV | BPF_ALU (0xbc) should already do zero-extension.